test/framework/src/main/java/org/elasticsearch/test/ESSingleNodeTestCase.java

```diff
@@ -190,6 +190,7 @@ private Node newNode() {
             .put(EsExecutors.PROCESSORS_SETTING.getKey(), 1) // limit the number of threads created
             .put("transport.type", getTestTransportType())
             .put(Node.NODE_DATA_SETTING.getKey(), true)
+            .put(TestZenDiscovery.USE_ZEN2.getKey(), randomBoolean())
```

Comment:
In the other tests that are migrated to use Zen2 we set this to `true` (i.e. we are not testing the Zen1 case any more). I think that's what we want to do here too, but in any case we should be consistent about this.

Reply:
I agree that we should be consistent about this. For now, I've switched this to always use Zen2 (see f9fb7df), but will revisit once we migrate more integration tests.